components/GlobalStyles.js

```diff
@@ -25,10 +28,11 @@ export const theme = {
     cardfield: '#77D9A1',
```

Comment:
I don't think `cardfield` is a CSS property!?

Reply:
true, it is not a property. This is also a variable name for using the color :)